### PR TITLE
(fixed) added 64x64 favicon

### PR DESCRIPTION
**Pull request by soh3e**
Changed the stock favicon to the AEC logo; had to take the bottom text out because the larger height makes it a rectangle shape rather than a square (what a favicon should be)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <link rel="icon" href="/src/logo.ico" type="image/x-icon">
```
Comment (nasirhm) [inline index.html:5]: The updated logo has the modified logo, the rocket is a part of the logo. 
Can you please update the `.ico` to have the whole logo.

File: src/site.webmanifest [hunk 2]
```diff
@@ -0,0 +1 @@
+{"name":"","short_name":"","icons":[{"src":"/android-chrome-192x192.png","sizes":"192x192","type":"image/png"},{"src":"/android-chrome-512x512.png","sizes":"512x512","type":"image/png"}],"theme_color":"#ffffff","background_color":"#ffffff","display":"standalone"}
```
Comment (nasirhm) [inline src/site.webmanifest]: We don't make use of `webmanifest` at the moment. Can we please remove it.

**Review by nasirhm — CHANGES_REQUESTED** [review]
Can you please update the Text along the favicon with `Applied Engineering Club @ FJC`.

Would you like to incorporate the banner removal & attribution addition changes in this PR ?

File: index.html (hunk 1 above)
Comment (soh3e) [reply index.html:5]: Updated the .ico to have the rocket within the logo, but not the text for aesthetic reasons

**Review by nasirhm — APPROVED** [review]
Looks Good to Me.

**Pull request merged**